x-pack/legacy/plugins/monitoring/public/components/elasticsearch/node_detail_status/index.js

```diff
@@ -45,9 +48,9 @@ export function NodeDetailStatus({ stats }) {
     },
     {
       label: i18n.translate('xpack.monitoring.elasticsearch.nodeDetailStatus.freeDiskSpaceLabel', {
-        defaultMessage: 'Free Disk Space'
+        defaultMessage: 'Free Disk Space',
       }),
-      value: formatMetric(freeSpace, '0.0 b'),
+      value: formatMetric(freeSpace, '0.0 b') + ' (' + formatMetric(percentSpaceUsed, '0,0.[00]', '%') + ')',
```

Comment:
By default, this will add a space between the number and the `%` sign, which I think looks weird. I'd recommend `formatMetric(percentSpaceUsed, '0,0.[00]', '%', { prependSpace: false })` instead

Reply:
Good eye! Thanks!